schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals(
+	id INT,
```

Comment:
- [ Optional ] Kindly I recommend you to make the id column not null and auto-incremented and mark the id column as primary key Just like this : -
   ` id int NOT NULL IDENTITY(1,1),     PRIMARY KEY (id)`